meeting_minutes/2021-11-23.md

```diff
@@ -0,0 +1,33 @@
+# working group meeting on 2021-11-23
+
+The agenda is in the [Kanban board](https://github.com/llvm/llvm-iwg/projects/1)
+
+## Participants
+
+* Tom Stellard
+* Anton Korobeynikov
+* Mehdi Amini
+* Reid Kleckner
+* Kamau Bridgeman
+
+## A Request for Comment on Code Review Process #73
+
+* The summary has been completed
+* Need reviews on the PR to https://github.com/llvm/llvm-iwg/pull/82/
+* Needs to be finalize and approved then send to the board this week.
+
+## Bugzilla migration
+
+* Migration effort needs more support and help.
+* GitHub has supplied dedicated support engineers to help migration over the thanksgiving weekend.
+
+## Conversation on how to continue to formalize and structure IWG items
+
+* Mentorship program could be a testing ground to support this.
```

Comment:
Does the term `mentorship` here means https://mentorship.lfx.linuxfoundation.org/ or more generally?

Reply:
Thank you for the review and approval, this is referring to and was sparked by conversations at the recently concluded llvm developer conference 2021 about starting such an initiative in the LLVM community by the LLVM foundation.